### PR TITLE
Disassembler Dupe Blacklist

### PR DESCRIPTION
**Pull request by Connor-Colenso**
More exploits from this very useful machine...

### CODE REVIEW

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 1]
```diff
@@ -1719,6 +1752,15 @@ private void makePlasmaForgeRecipes_DTPF() {
                         new FluidStack[]{Materials.DimensionallyTranscendentResidue.getFluid(1000)},
                         40*20, 128_000_000, eternal_heat);
 
+
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1755]: why you ship this in the dissasembler pr ?

File: dependencies.gradle [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // Add your dependencies here
 
 dependencies {
+    compile("com.github.GTNewHorizons:GT5-Unofficial:5.09.40.96-pre:dev")
```
Comment (mitchej123) [inline dependencies.gradle:4]: Two versions?

Comment (mitchej123) [reply dependencies.gradle:4]: Also no pre's!

**Comment by Dream-Master** [comment]
?

**Pull request closed without merging**